client/src/components/game/menu/OptionsForm.vue

```diff
@@ -138,19 +138,103 @@
         </select>
       </div>
     </div>
-    
+
+    <div class="row bg-secondary pt-1 pb-1">
+      <label for="" class="col col-form-label">Background</label>
+    </div>
+
+    <div class="row bg-secondary pt-1 pb-1 ml-1">
+      <label for="nebula-frequency" class="col col-form-label">Nebula Frequency</label>
+      <div class="col">
+        <input type="number" min="0" max="16" step="1" class="form-control" id="nebula-frequency" v-model="settings.map.background.nebulaFrequency" :disabled="isSavingSettings">
+      </div>
+    </div>
+
+    <div class="row bg-secondary pt-1 pb-1 ml-1">
+      <label for="nebula-density" class="col col-form-label">Nebula Density</label>
+      <div class="col">
+        <input type="number" class="form-control" id="nebula-density" v-model="settings.map.background.nebulaDensity" :disabled="isSavingSettings">
+      </div>
+    </div>
+
+    <div class="row bg-secondary pt-1 pb-1 ml-1">
+      <label for="nebula-opacity" class="col col-form-label">Nebula Opacity</label>
+      <div class="col">
+        <input type="number" min="0.0" max="1.0" step="0.125" class="form-control" id="nebula-opacity" v-model="settings.map.background.nebulaOpacity" :disabled="isSavingSettings">
+      </div>
+    </div>
+
+    <div class="row bg-secondary pt-1 pb-1 ml-1">
+      <label for="stars-opacity" class="col col-form-label">Stars Opacity</label>
+      <div class="col">
+        <input type="number" min="0.0" max="1.0" step="0.125" class="form-control" id="stars-opacity" v-model="settings.map.background.starsOpacity" :disabled="isSavingSettings">
+      </div>
+    </div>
+
     <div class="row bg-secondary pt-1 pb-1">
-      <label for="nebulaDensity" class="col-12 col-sm-6 col-form-label">Nebula Density</label>
+      <label for="background-blend-mode" class="col-12 col-sm-6 col-form-label">Blend Mode</label>
       <div class="col-12 col-sm-6">
-        <select class="form-control" id="nebulaDensity" v-model="settings.map.nebulaDensity" :disabled="isSavingSettings">
-          <option value="none">None</option>
-          <option value="sparse">Sparse</option>
-          <option value="standard">Standard</option>
-          <option value="abundant">Abundant</option>
+        <select class="form-control" id="background-blend-mode" v-model="settings.map.background.blendMode" :disabled="isSavingSettings">
+          <option value="ADD">ADD</option>
```

Comment:
Can this be renamed to something a normie can read?

Reply:
haha XD, humm smooth and dynamic maybe? normal and boosted?

Reply:
I have no idea what they are so I'll leave it to your best judgement.